exporter_test.go

```diff
@@ -1074,6 +1074,21 @@ version = "4.5.6"
 				h.AssertNil(t, err)
 				h.AssertContains(t, fakeAppImage.SavedNames(), append(opts.AdditionalNames, fakeAppImage.Name())...)
 			})
+
+			it("should display that the SBOM is missing", func() {
```

Comment:
Do we also need a test that when the SBOM is not missing, it is copied?

Reply:
In this test (exporter_test.go), the SBOM is missing because the SBOMs have not yet been generated. 

The alternative test case has been implemented in the acceptance test ( acceptance/exporter_test.go ) where fake SBOMs have been copied ( copyFakeSboms(t) ) in the out directory.
We have to copy the fake SBOMs before this call: _exportTest.Start(t, updateAnalyzedTOMLFixturesWithRegRepoName)_ because there is a call to _h.MakeAndCopyLifecycle()_ where all the files available in the _out_ directory are copied in new empty destination directory (_containerBinaryDir_).
Then the docker image is built (_h.DockerBuild_) from the _out_ directory which contains _binary and SBOM_s. The SBOMs are successfully packaged in the /cnb/lifecycle/ folder and we check the output "Copying launcher SBOM (sbom.syft.json)"


So with these two tests, we are covering the both test cases (SBOM is _available_ and SBOM is _missing_).